app/aggregates/reviewing/review.rb

```diff
@@ -26,6 +27,7 @@ def receive_application(application_id:, correlation_id: nil)
     end
 
     def send_back(application_id:, user_id:, reason:)
+      raise NotReceived unless received?
       raise AlreadySentBack if @state.equal?(:sent_back)
```

Comment:
Just thinking it might be good to wrap these other guard clause conditionals in methods like `received?`

Reply:
Handled in next pr.